### PR TITLE
Separate data layers

### LINKED ISSUE #35: Separating feature layers
Working on frontend bug fix with all layers being toggled off.

### PR DESCRIPTION
**Pull request by n3a9**
Resolves #35 

- Separate data layers to separate buttons for independent toggle
- Set specific colors per layer
- Initial implementation for bottom tab bar (*detached from panel*)

### CODE REVIEW

File: c2tc-mobile/components/Tab.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    return this.state.page;
+  }
+
+  render() {
+    return (
+      <Tabs
+        selected={this.state.page}
+        style={{ backgroundColor: "white", opacity: 0.5 }}
+        selectedStyle={{ color: "purple" }}
+        onSelect={el => this.setState({ page: el.props.name })}
```
Comment (choiboy98) [inline c2tc-mobile/components/Tab.js]: Create a different method for this since it is faster.

File: c2tc-mobile/components/Tab.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      >
+        <Text
+          name="filter"
+          selectedIconStyle={{ borderTopWidth: 2, borderTopColor: "purple" }}
```
Comment (choiboy98) [inline c2tc-mobile/components/Tab.js]: create a new stylesheet please

File: c2tc-mobile/components/Tab.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        >
+          Filters
+        </Text>
+        <Text
+          name="contact"
+          selectedIconStyle={{ borderTopWidth: 2, borderTopColor: "purple" }}
```
Comment (choiboy98) [inline c2tc-mobile/components/Tab.js]: repeated code!!! If you had stylesheet, this wouldn't have happened. smh

File: c2tc-mobile/screens/LiveLocation.js [hunk 4]
```diff
@@ -12,17 +12,13 @@ const LATITUDE_DELTA = 0.0922;
...
-const layer2Data = require("../assets/data/layer2_loc.json");
-const layerData = [policeLocations, layer2Data];
-const layerKey = ["police", "light", "construction"];
-const policeColor = "#841584";
+const lightLocations = require("../assets/data/light_locations.json");
 
-let id = 0;
-let renderLayers = true;
+const layerData = { police: policeLocations, lights: lightLocations };
+const colorData = { police: "#841584", lights: "#000000" };
```
Comment (choiboy98) [inline c2tc-mobile/screens/LiveLocation.js]: put the color in a different variable to make code nicer

File: c2tc-mobile/components/Tab.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+  getState() {
+    return this.state.page;
+  }
+
+  render() {
+    return (
+      <Tabs
+        selected={this.state.page}
+        style={{ backgroundColor: "white", opacity: 0.5 }}
```
Comment (meghatron3000) [inline c2tc-mobile/components/Tab.js]: put the style on a style sheet?

File: c2tc-mobile/components/Tab.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+import Tabs from "react-native-tabs";
+
+export default class Tab extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { page: "filter" };
+  }
+
+  getState() {
```
Comment (meghatron3000) [inline c2tc-mobile/components/Tab.js]: what exactly are you returning? change your function name to be more descriptive.

**Pull request merged**